### PR TITLE
ignore filter for pandas

### PR DESCRIPTION
**Pull request by nabobalis**
Fixes https://github.com/sunpy/sunpy/issues/7390

Lets see if I got this correct.

### CODE REVIEW

**Comment by nabobalis** [comment]
docfail is real:

`/home/runner/work/sunpy/sunpy/.tox/build_docs/lib/python3.9/site-packages/sunpy/timeseries/timeseriesbase.py:docstring of sunpy.timeseries.timeseriesbase.GenericTimeSeries.to_array:3: WARNING: py:obj reference target not found: pandas.DataFrame.to_numpy`

File: sunpy/timeseries/timeseriesbase.py [hunk 1]
```diff
@@ -855,7 +855,7 @@ def to_array(self, **kwargs):
         Parameters
         ----------
         **kwargs : `dict`
-            All keyword arguments are passed to `pandas.DataFrame.to_numpy`.
+            All keyword arguments are passed to ``pandas.DataFrame.to_numpy``.
```
Comment (dstansby) [inline sunpy/timeseries/timeseriesbase.py:858]: This looks like a bug in the pandas docs? Go to https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html, find `to_numpy` and there's no link there. So i think worth reporting as a bug to them?

Comment (dstansby) [reply sunpy/timeseries/timeseriesbase.py:858]: Done at https://github.com/pandas-dev/pandas/issues/57016

**Review by dstansby — APPROVED** [review]

File: sunpy/timeseries/timeseriesbase.py (hunk 1 above)
Comment (nabobalis) [reply sunpy/timeseries/timeseriesbase.py:858]: Thanks

**Pull request merged**